test/components/error_summary_test.rb

```diff
@@ -0,0 +1,84 @@
+require 'component_test_helper'
+
+class ErrorSummaryTest < ComponentTestCase
+  def component_name
+    "error-summary"
+  end
+
+  test "fails to render no data is given" do
```

Comment:
Is this test correct? Looking at the component template I can't see a check for any of the passed data, looks like it'll render empty elements regardless?